src/algebra/pointwise.lean

```diff
@@ -132,7 +132,8 @@ instance [semigroup α] : semigroup (set α) :=
 
 @[to_additive set.add_monoid]
 instance [monoid α] : monoid (set α) :=
```

Comment:
Ideally we'd have this instance for the new `mul_one_class (set α)` too

Reply:
Done.